crates/languages/src/css/highlights.scm

```diff
@@ -71,7 +74,7 @@
   (float_value)
 ] @number
 
-(unit) @type
+(unit) @number.unit
```

Comment:
I can't find this capture used in any other grammar on GitHub: https://github.com/search?q=%22@number.unit%22&type=code. I would keep it as type.
```suggestion
(unit) @type
```

Reply:
I think adding more capture groups would be better. (Currently there are a few categories of capture groups). If zed members agree with this change, I can also add it to the documentation. In any case, numbers are not types. This is also what VSCode and Helix do. I think following the behavior of other editors is better.

Reply:
> This is also what VSCode and Helix do.

Helix does not have a highlight query for `unit` at all, so not sure what you mean by that. VSCode uses `
keyword.other.unit` for units, so if anything let's go with `@keyword`.

Reply:
In helix, the unit color is inherited from the number, and in vscode, the unit has a separate color. So I opted for a compromise: the default inherits from numbers, but can be configured individually.

I changed it to `@keyword` as you suggested, but I wish we had a capture like `@number.unit` or `@keyword.unit`.